client/src/modules/reports/generate/account_report_multiple/account_report_multiple.html

```diff
@@ -0,0 +1,62 @@
+<bh-report-preview
+  ng-if="ReportConfigCtrl.previewGenerated"
+  source-document="ReportConfigCtrl.previewResult"
+  on-clear-callback="ReportConfigCtrl.clearPreview()"
+  on-save-callback="ReportConfigCtrl.requestSaveAs()">
+</bh-report-preview>
+
+<div ng-show="!ReportConfigCtrl.previewGenerated">
+  <div class="row">
+    <div class="col-md-12">
+      <h3 class="text-capitalize" translate>REPORT.REPORT_ACCOUNTS_MULTIPLE.TITLE</h3>
+      <p class="text-info" translate>REPORT.REPORT_ACCOUNTS_MULTIPLE.DESCRIPTION</p>
+    </div>
+  </div>
+
+  <div class="row" style="margin-top : 10px">
+    <div class="col-md-6">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span translate>REPORT.UTIL.OPTIONS</span>
+        </div>
+
+        <div class="panel-body">
+          <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate>
+
+            <bh-account-select-multiple
+              id="account-id"
+              account-ids="ReportConfigCtrl.accountIds"
+              label="FORM.SELECT.ACCOUNTS"
+              name="account"
+              on-select-callback="ReportConfigCtrl.selectAccount(account)"
+              on-change="ReportConfigCtrl.onChangeAccounts(id)"
+              exclude-title-accounts="true"
+              required="true"
+              validation-trigger="ConfigForm.$submitted">
+            </bh-account-select-multiple>
+
+            <!-- Date interval  -->
+            <!-- @TODO this should use a component that callback with well defined dates -->
+            <bh-date-interval
+              date-from="ReportConfigCtrl.reportDetails.dateFrom"
+              date-to="ReportConfigCtrl.reportDetails.dateTo"
+              required="true"
+              validation-trigger="ConfigForm.$submitted">
```

Comment:
It looks like [validationTrigger was removed from bh-date-interval](https://github.com/IMA-WorldHealth/bhima-2.X/commit/2e8ebe0910ae7df77231c464c09d63dbcb10f59d#diff-33e4c895023c285d3bf777adbe56d375).  Please remove it here.